backend/xivdpscalc/pps/sch_action.py

```diff
@@ -0,0 +1,39 @@
+""" Sch specific enums for simulation """
+
+from enum import Enum, auto
+from xivdpscalc.types import CastTime, ElapsedTime
+
+class SchAction(Enum):
+    """
+    Enum representing possible sch actions
+    """
+    BROIL3 = (0, 290, 2.5, True, False, 0)
+    RUIN2 = (1, 200, 0, True, False, 0)
+    AETHERFLOW = (2, 0, 0, False, False, 60)
+    ENERGYDRAIN = (3, 100, 0, False, False, 3)
+    BIOLYSIS = (4, 70, 0, True, True, 0)
+    DISSIPATION = (5, 0, 0, False, False, 180)
+    SWIFTCAST = (6, 0, 0, False, False, 60)
+    CHAINSTRATAGEM = (7, 0, 0, False, False, 120)
+
+    def __init__(self, index: int, potency: int, cast_time: CastTime, is_gcd: bool, is_dot: bool, cooldown: ElapsedTime):
```

Comment:
Might be worth telling pylint to ignore this with a `# pylint: disable=too-many-arguments`